tools/spec.c

```diff
@@ -276,6 +289,70 @@ void parse_rom_spec(char *spec, struct Segment **segments, int *segment_count)
     }
 }
 
+/**
+ * @brief Parses the spec, looking only for the segment with the name `segmentName`.
+ * Returns true if the segment was found, false otherwise
+ * 
+ * @param[out] dstSegment The Segment to be filled. Will only contain the data of the searched segment, or garbage if the segment was not found. dstSegment must be previosly allocated, a stack variable is recommended
```

Comment:
Wouldn't setting it to `NULL` be better if none is found?

Reply:
No, because modifying the local copy of the pointer (the pointer itself, not the values pointed by the pointer) won't affect the value the callee has

Reply:
Maybe write zeros or something then. I suppose the point is that you have to use the return value.